LEGEND.md

```diff
@@ -24,31 +24,33 @@
 
 #### <a name="macos-slides">macOS:</a>:
 
-| Combination | Description                         |
-| ----------- | ----------------------------------- |
-| cmd + 1     | Waheguru Slide                      |
-| cmd + 2     | Mool Mantra Slide                   |
-| cmd + 3     | Empty Slide                         |
-| cmd + 4     | Anand Sahib (Bhog)                  |
-| cmd + 5     | Help Guide                          |
-| cmd + 6     | Shortcut Legend                     |
-| cmd + /     | Jump to search bar                  |
-| ----------- | -----------                         |
-| cmd + g     | Add Dhan Guru slide or announcement |
+| Combination                                | Description                                 |
+| ------------------------------------------ | ------------------------------------------- |
+| cmd + 1                                    | Waheguru Slide                              |
+| cmd + 2                                    | Mool Mantra Slide                           |
+| cmd + 3                                    | Empty Slide                                 |
+| cmd + 4                                    | Anand Sahib (Bhog)                          |
+| cmd + 5                                    | Help Guide                                  |
+| cmd + 6                                    | Shortcut Legend                             |
+| cmd + /                                    | Jump to search bar                          |
+| cmd + g                                    | Add Dhan Guru slide or announcement         |
+| spacebar (when in a shabad)                | Switch between main line and unchecked line |
+| spacebar (when a waheguru slide is active) | switch back to shabad                       |
```

Comment:
@maneetpaul can you approve the copy for the text? should probably keep capitalization consistent

Reply:
Yes - need to capitalize the "s" in the last switch. And also all the ctrl and cmds should be caps CTRL CMD